### ISSUE TITLE
bz_LoadPhy uses base 1 instead of base 0 for labeling channels

### ISSUE THREAD

**Issue by raswanson** [buzsakilab/buzcode#264]
Hey guys - just flagging the fact that:
- bz_LoadPhy uses base 1 for identification of the channel of max waveform for each unit (spikes.maxWaveformCh)
- bz_GetLFP uses base 0 for channel identification (a la neuroscope)
- bz_GetSpikes appears to use base 0 (please confirm @DavidTingley)

We should visit this to make sure all are the same.

**Comment by brendonw1** [comment]
Basically going forward it of course makes sense to make bz_LoadPhy use base0 to match.  But @AntonioFR8 and others who have already used bz_LoadPhy will need to find a way to make this work for them

**Comment by raswanson** [comment]
Yeah exactly. We were thinking we'd just have to chat with Antonio/Manu tomorrow to see how much that would break :P
